src/block/column/editor.scss

```diff
@@ -4,8 +4,13 @@
 	z-index: 2;
 }
 
-.editor-styles-wrapper .stk-column-wrapper > .block-editor-inner-blocks {
-	width: 100%;
+.editor-styles-wrapper {
+	.stk-block.stk-block-column .stk-column-wrapper {
```

Comment:
@vmc08 question here, the new selector uses `.stk-block.stk-block-column`, this limits the styles to only the column block, so now the `width: 100%` might not apply to other blocks that might use `.stk-column-wrapper` (I think the Feature block or Feature Grid block uses them). Is this okay?